app/assets/sass/patterns/_step-by-step-navigation.scss

```diff
@@ -5,7 +5,7 @@
 .app-step-nav-header {
   position: relative;
   padding: 10px;
-  background: govuk-colour("grey-4");
+  background: govuk-colour("light-grey");
```

Comment:
I wonder if we want to pass `$legacy` attributes here so that users can opt to use the legacy palette and this won't blow up?